pkg/cloudprovider/providers/gce/gce_disks.go

```diff
@@ -58,6 +58,9 @@ const (
 	diskSourceURITemplateRegional   = "%s/regions/%s/disks/%s" //{gce.projectID}/regions/{disk.Region}/disks/repd"
```

Comment:
Check with @davidz627 to make sure he does what this PR is doing for the CSI driver.

Reply:
https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/issues/109

I originally thought we didn't need to do this for the CSI Driver as we only support dynamic provisioning. However, to support migration we will need to support this case